src/molecules/card-workout/index.jsx

```diff
@@ -7,7 +7,7 @@ import Stack from "@mui/material/Stack";
 const CardWorkout = () => {
 	const { data } = useGet(endpoints.exercises);
 	return (
-		<div>
+		<div className="a-class">
```

Comment:
what is this doing?

Reply:
Tells me where I can put my styling element later!